bimmer_connected/vehicle_status.py

```diff
@@ -143,6 +143,20 @@ def gps_position(self) -> (float, float):
         pos = self._state.attributes[SERVICE_STATUS]['position']
         return float(pos['lat']), float(pos['lon'])
 
+    @property
+    @backend_parameter
+    def gps_heading(self) -> (int):
+        """Get the last known heading of the vehicle.
+
+        This only provides data, if the vehicle tracking is enabled!
+        """
+        pos = self._state.attributes[SERVICE_STATUS]['position']
+        if not self.is_vehicle_tracking_enabled:
+            _LOGGER.warning('Positioning status is %s', pos['status'])
+            return None
+        pos = self._state.attributes[SERVICE_STATUS]['position']
```

Comment:
This line is already in 153. I see the same goes for lines 143 and 139.

Reply:
True, but refactoring doesn’t makes it easier to read though. Did some minot changes